krux/cli.py

```diff
@@ -274,6 +292,12 @@ def add_logging_args(parser):
         choices=LEVELS.keys(),
         help='Verbosity of logging. (default: %(default)s)'
     )
+    group.add_argument(
+        '--log-file',
+        default=None,
```

Comment:
i believe there's some way to validate that what you pass in is a file path - can we add that here?